public/app/core/utils/fetch.ts

```diff
@@ -105,13 +105,15 @@ export async function parseResponseBody<T>(
         return response.blob() as any;
 
       case 'json':
-        try {
-          return await response.json();
-        } catch (err) {
-          console.warn(`${response.url} returned an invalid JSON -`, err);
+        // An empty string is not a valid JSON.
```

Comment:
This works for the Graphite Plugin, since we now request a response type of text, where it used to default to json. https://github.com/grafana/grafana/pull/48594